spec/rails_helper.rb

```diff
@@ -78,4 +78,8 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.render_views
+
+  config.example_status_persistence_file_path = 'specs.txt'
```

Comment:
usually we keep such files in the `tmp` folder - then there should be no need to add that to gitignore